Resources/ArmTemplates/template.json

```diff
@@ -266,6 +266,18 @@
         },
         "PreviewRedirectUri:1:ContentType": {
             "type": "string"
+        },
+        "PreviewRedirectUri:2:RedirectUri":{
```

Comment:
2 - footer
3 - jobsector
etc. this number is not good name conversion. We need give the name more meaningful.